rest_framework/compat.py

```diff
@@ -33,6 +33,14 @@ def unicode_to_repr(value):
     return value
 
 
+def total_seconds(timedelta):
+    # TimeDelta.total_seconds() is only available in Python 2.7
+    if hasattr(timedelta, 'total_seconds'):
+        return timedelta.total_seconds()
+    else:
+        return (timedelta.microseconds + (timedelta.seconds + timedelta.days * 24 * 3600) * 10 ** 6) / 10 ** 6
```

Comment:
I think it might be clearer if only the microsecond portion got divided by 1000000. Eg.

   (timedelta.days * 86400) + timedelta.seconds + (timedelta.microseconds / 1000000.0)